third_party/Makefile

```diff
@@ -81,12 +80,7 @@ rocm_rocblas: sb_micro_path
 ifeq (, $(wildcard $(SB_MICRO_PATH)/bin/rocblas-bench))
 	if [ -d rocBLAS ]; then rm -rf rocBLAS; fi
 	git clone -b ${ROCM_VERSION} https://github.com/ROCmSoftwarePlatform/rocBLAS.git ./rocBLAS
-ifeq (${ROCM_VERSION}, rocm-4.0.0)
-	sed -i '/CMAKE_MATCH_1/a\      get_filename_component(HIP_CLANG_ROOT "$${HIP_CLANG_ROOT}" DIRECTORY)'  /opt/rocm/hip/lib/cmake/hip/hip-config.cmake
-	cd ./rocBLAS && HIPCC_COMPILE_FLAGS_APPEND="-D_OPENMP=201811 -O3 -Wno-format-nonliteral -DCMAKE_HAVE_LIBC_PTHREAD -parallel-jobs=2" HIPCC_LINK_FLAGS_APPEND="-lpthread -O3 -parallel-jobs=2" ./install.sh -idc -a ${ROCM_ARCH}
-else 
 	cd ./rocBLAS && ./install.sh -idc
```

Comment:
could you build the client only `./install.sh --clients-only`?

currently you will rebuild the whole rocblas and downgrade the prebuilt /opt/rocm/rocblas inside container, it adds **12GB** to the image

Reply:
I tried, but it failed, so I can only try to build all.
![image](https://user-images.githubusercontent.com/37182275/132936650-6025334a-c745-4552-b89f-0c7f1104e0bf.png)